grabber/main.go

```diff
@@ -117,44 +119,50 @@ func listener(importer *backend.Backend) {
 	for {
 		select {
 		case <-ticker:
-			latestBlocknumber := getLatestBlockNumber(importer)
-			importer.Lgr.Debug("Getting block in listener", zap.Int64("Block Number", latestBlocknumber))
-			if prevHeader != latestBlocknumber {
-				importer.Lgr.Info("Getting block in listener", zap.Int64("Block Number", latestBlocknumber))
-				block, err := importer.BlockByNumber(latestBlocknumber)
-				if block != nil {
+			latest, err := importer.GetLatestBlockNumber()
+			if err != nil {
+				importer.Lgr.Error("Listener: Failed to get latest block number", zap.Error(err))
+				continue
```

Comment:
All the fatals that were here retry on the next tick now, so the other processes aren't disrupted.